src/content/doc-surrealdb/installation/upgrading/2.0.mdx

```diff
@@ -7,14 +7,22 @@ description: This guide will help you upgrade your current SurrealDB installatio
 
 # Upgrading to `2.0.0` 
 
-The `2.0.0` release of SurrealDB includes many new features, improvements, and bug fixes. However, because of changes in the underlying way that SurrealDB stores data, three steps are required to migrate to `2.0.0`. We've listed them below, and we advise following this migration guide whether or not you encounter this issue!
+The `2.0.0` release of SurrealDB includes [many new features, improvements, and bug fixes](https://surrealdb.com/releases#v2-0-0). However, because of changes in the underlying way that SurrealDB stores data, three steps are required to migrate to `2.0.0`. 
+
+We've listed them below, and we advise following this migration guide whether or not you encounter this issue!
 
 ## Issues
```

Comment:
I would also probably add the following issues:
- Authentication is now enabled by default as you previously would with `--auth`. The `--unauthenticated` flag is now required in order to provide the previous default behavior.
- Headers used to communicated via HTTP with SurrealDB now require the `surreal-` prefix. For example, the legacy `ns` and `db` headers are now `surreal-ns` and `surreal-db`.

Reply:
Ah yes!

Reply:
Also the following could also be considered breaking changes since 1.x:
- Specifying the level on which credentials will be authenticated is now required when connecting to SurrealDB. By default, this level will be root. This can be provided with the --auth-level flag in the CLI or the surreal-auth-ns and surreal-auth-db headers in the HTTP REST API.
- SurrealDB now listens only for connections from the local machine unless another interface (e.g. `0.0.0.0`) is provided via the `--bind` command line argument.
- SurrealDB now does not print secrets in response to `INFO` statements. The values of the secrets will appear as `[REDACTED]` to prevent accidental leakage. The export functionality will still print the values of the secrets. An `UNREDACTED` clause will be added soon to provide the previous behavior.

Reply:
Even if mostly addressed by the `fix` command, it may be worth mentioning the changes from `DEFINE TOKEN` and `DEFINE SCOPE` to `DEFINE ACCESS` here as well.

Reply:
Also, I think we made changes to the parser that should probably be noted here, right?

Reply:
So I have updated this to be Breaking changes and included most of the breaking changes. I think the surreal fix command does fix most of these but have added all the ones listed here. 

@gguillemas I linked to the release notes RE parser updates